src/PrestaShopBundle/Form/Admin/Sell/Product/Options/OptionsType.php

```diff
@@ -56,7 +56,7 @@ public function configureOptions(OptionsResolver $resolver)
         parent::configureOptions($resolver);
         $resolver->setDefaults([
             'required' => false,
-            'label' => false,
+            'label' => $this->trans('Options', 'Admin.Catalog.Feature'),
```

Comment:
Same here, the string already exists in Admin.Global

Reply:
Same comment as for Descriptions https://github.com/PrestaShop/PrestaShop/pull/28752#discussion_r937621147 I think the generic wording and the tab names are two different notions that's why it justifies having two different namespaces